trashtalk/fixtures/cleanups.json

```diff
@@ -0,0 +1 @@
e": true, "category": 6}}, {"model": "cleanups.tool", "pk": 30, "fields": {"name": "Weed Wrench", "description": "Pulls out woody plants such as French broom; Easiest to use with soft and slightly moist soils", "image_static_location": "images/icons/weed_wrench.png", "available": true, "category": 6}}, {"model": "cleanups.tool", "pk": 31, "fields": {"name": "Grass Trimmers", "description": "Whip back and forth to cut tall grasses and weeds", "image_static_location": "images/icons/grass_trimmer.png", "available": true, "category": 6}}, {"model": "cleanups.tool", "pk": 32, "fields": {"name": "Hula Hoe", "description": "Cuts weeds in sandy soils", "image_static_location": "images/icons/hula_hoe.png", "available": true, "category": 6}}, {"model": "cleanups.tool", "pk": 33, "fields": {"name": "Standard Hoe", "description": "Cuts weeds just under the surface of the soil", "image_static_location": "images/icons/hoe.png", "available": true, "category": 6}}, {"model": "cleanups.tool", "pk": 34, "fields": {"name": "Mattock Tiller", "description": "A heavy duty tool for weeding and breaking up soil. One side is a hoe and the other side is a fork", "image_static_location": "images/icons/mattock_tiller.png", "available": true, "category": 6}}, {"model": "cleanups.tool", "pk": 35, "fields": {"name": "Skimmer", "description": "Collects debris from water", "image_static_location": "images/icons/skimmer.png", "available": true, "category": 7}}, {"model": "cleanups.tool", "pk": 36, "fields": {"name": "Leaf Rakes", "description": "Shaped like a fan and used to rake leaves", "image_static_location": "images/icons/rakes.jpg", "available": true, "category": 7}}, {"model": "cleanups.tool", "pk": 37, "fields": {"name": "Bow Rake", "description": "Rakes hard materials or moves through roots and ruts", "image_static_location": "images/icons/bow_rake.jpg", "available": true, "category": 7}}, {"model": "cleanups.tool", "pk": 38, "fields": {"name": "McLeod", "description": "Combines a large hoe-like blade on one side for cutting and a large rake on the other side. Useful for clearing large areas of vegetation and building trails", "image_static_location": "images/icons/mcleod.png", "available": true, "category": 7}}, {"model": "cleanups.tool", "pk": 39, "fields": {"name": "Scoop Shovel", "description": "Scoops large amounts of material; also called a snow shovel", "image_static_location": "images/icons/scoop_shovel.png", "available": true, "category": 7}}, {"model": "cleanups.tool", "pk": 40, "fields": {"name": "Flat Shovel", "description": "Scoops soil or moves piles; Not recommended for digging holes", "image_static_location": "images/icons/flat_shovel.png", "available": true, "category": 7}}, {"model": "cleanups.tool", "pk": 41, "fields": {"name": "Pitch Fork", "description": "Moves or picks up loose materials such as hay, compost, or vegetation. Also spreads straw or mulch, etc.", "image_static_location": "images/icons/pitch_fork.png", "available": true, "category": 7}}]
```

Comment:
1 . One of the tools has a misspelled image location. 
     a. I can't remember which one. However, it's more obvious when looking at a rendering of the tools
2 . There needs to be a way to account for the different sizes of some tools are available it. 

I address these issues in my own fixture: tools.json. I will find a venue to share this file.

Reply:
Sure. At some point, we should probably do a once over and update these tools in the Django Admin, and then generate another fixtures file for all the tools.